lgc/elfLinker/ElfLinker.cpp

```diff
@@ -556,8 +556,8 @@ bool ElfLinkerImpl::link(raw_pwrite_stream &outStream) {
     if (sectionIndex == noteSectionIdx)
       continue;
     OutputSection &outputSection = m_outputSections[sectionIndex];
-    unsigned align = std::min(unsigned(outputSection.getAlignment()), 4U);
-    outStream << StringRef("\0\0\0", 3).slice(0, -outStream.tell() & align - 1);
```

Comment:
> Seems like some offset computation changed and leads to a test failure.
Right, this computation used to go wrong when `align` was 0, which should have been treated the same as 1, but actually made this code always insert 3 padding bytes.